Userland/Libraries/LibMarkdown/CodeBlock.cpp

```diff
@@ -102,21 +54,7 @@ OwnPtr<CodeBlock> CodeBlock::parse(Vector<StringView>::ConstIterator& lines)
     if (!line.starts_with(tick_tick_tick))
         return {};
 
-    // Our Markdown extension: we allow
-    // specifying a style and a language
-    // for a code block, like so:
-    //
-    // ```**sh**
-    // $ echo hello friends!
-    // ````
-    //
-    // The code block will be made bold,
-    // and if possible syntax-highlighted
-    // as appropriate for a shell script.
```

Comment:
It's used quite frequently in the manpages (e.g. https://raw.githubusercontent.com/SerenityOS/serenity/master/Base/usr/share/man/man1/Terminal.md) though surprisingly, I see more `**foo` than `**foo**`...curious

Reply:
This extension does not seem useful enough to merit breaking syntax highlighting in github. Also If we want to support both `**foo` and `**foo**` (which worked before due to a quirk of the old parser) we would have to specially parse it, which doesn't seem worth it to me. So I think there are three options here, in order of how much I want to do them:

1) Remove all `**foo`s from our man pages
2) Pick one of `**foo` and `**foo**` to be the correct usage of our extension
3) Re-add support for both cases.

Reply:
fwiw github seems to allow `**foo**` (and `**foo`) as language identifiers there, so perhaps we could follow GH and allow that, and parse these in a later stage (e.g. recognize `**`, `*`, etc. in `Code::Code()` and set a style accordingly).

I don't think these are particularly troublesome, not sure if making LibMarkdown comply with one of the...many many markdown specs is a goal here

Reply:
Personally, I don't have a strong opinion on which one to choose, as long as Serenity's Markdown files can still be parsed by it. :P

If you want a tie-breaker from me, I would expect this: ` ```sh ` (Because what are the asterisks even doing there?)

Reply:
The extension was re-added. Now it just uses a regex to parse the emphasis.